packages/alfa-css/test/properties/overflow/property.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { lex } from "@siteimprove/alfa-lang";
+import { test } from "@siteimprove/alfa-test";
+import { Alphabet } from "../../../src/alphabet";
+import { overflow } from "../../../src/properties/overflow/property";
+import { overflowX } from "../../../src/properties/overflow/property-x";
+import { overflowY } from "../../../src/properties/overflow/property-y";
+import { Values } from "../../../src/values";
+
+const { keyword } = Values;
+
+test("Can parse full shorthand", t => {
+  const lexer = lex("clip hidden", Alphabet);
+  t.deepEqual(overflow.parse(lexer.result), {
+    overflowX: keyword("clip"),
+    overflowY: keyword("hidden")
+  });
+});
+
+test("Can parse short shorthand", t => {
+  const lexer = lex("clip", Alphabet);
+  t.deepEqual(overflow.parse(lexer.result), {
+    overflowX: keyword("clip"),
+    overflowY: keyword("clip")
+  });
+});
+
+test("Cannot parse invalid short shorthand", t => {
+  const lexer = lex("foo", Alphabet);
+  t.deepEqual(overflow.parse(lexer.result), {
+    overflowX: overflowX.initial(),
+    overflowY: overflowY.initial()
+  });
+});
+
+test("Cannot parse invalid full shorthand", t => {
+  const lexer = lex("foo bar", Alphabet);
+  t.deepEqual(overflow.parse(lexer.result), {
+    overflowX: overflowX.initial(),
+    overflowY: overflowY.initial()
+  });
+});
```

Comment:
Make a short helper for lexing and parsing things similar to this: https://github.com/Siteimprove/alfa/pull/116/files#diff-361044a82987e2ea39051466b46d8dbdR10